src/storage/src/hummock/sstable_store.rs

```diff
@@ -415,33 +437,18 @@ impl SstableStore {
             policy
         };
 
-        // future: fetch block if hybrid cache miss
-        let fetch_block = move |context: CacheContext| {
-            let range = range.clone();
-
-            async move {
-                let block_data = match store
-                    .read(&data_path, range.clone())
-                    .verbose_instrument_await("get_block_response")
-                    .await
-                {
-                    Ok(data) => data,
-                    Err(e) => {
-                        tracing::error!(
-                        "get_block_response meet error when read {:?} from sst-{}, total length: {}",
-                        range,
-                        object_id,
-                        file_size
-                    );
-                        return Err(anyhow::Error::from(HummockError::from(e)));
-                    }
-                };
-                let block = Box::new(
-                    Block::decode(block_data, uncompressed_capacity)
-                        .map_err(anyhow::Error::from)?,
-                );
-                Ok((block, context))
-            }
+        let fetcher = BlockFetcher::new(
```

Comment:
Does it mean that `BlockFetcher` will fetch a unit for every acquire?

Reply:
Yes for unit > 1.